components/eamxx/docs/developer/dev_testing/full_model_testing.md

```diff
@@ -0,0 +1,199 @@
+# Full-model Testing
+
+Full model system testing of EAMxx is done via CIME test cases
+(much like the rest of E3SM).
+
+We offer a number of test suites, including:
+
+- `e3sm_scream_v0`: Test the full set of V0 (pre-C++) tests
+- `e3sm_scream_v1`: Test the full set of V1 (C++) tests
+- `e3sm_scream_v1_at`: A smaller and quicker set of tests for autotesting (AT)
+- `e3sm_scream_hires`: A small number of bigger, longer-running
+(high-resolution) tests to measure performance
+
+```{ .shell .copy title="Running a Test Suite"}
+cd $repo/cime/scripts
+./create_test e3sm_scream_v1_at --wait
+```
+
+```{ .shell .copy title="Running a Single Test Case"}
+cd $repo/cime/scripts
+./create_test SMS.ne4_ne4.F2010-SCREAMv1 --wait
+```
+
+- There are many behavioral tweaks you can make to a test case, like
+changing the run length, test type, etc.
+      - Most of this is not specific to EAMxx and works for any CIME case.
+- This general information and much more can be found in the [CIME docs](http://esmci.github.io/cime/versions/master/html/users_guide/testing.html).
+
+## EAMxx-specific Model Configuration
+
+The main model-level configuration options for EAMxx are:
+
+1. ***grids***
+1. ***compsets***
+1. ***testmods***
+
+### Common EAMxx Grids[^grid_nomenclature]
+
+- `ne4_ne4`: low resolution
```

Comment:
In E3SM, "lowres" usually refers to ne30. We should prob stick with that, and call ne4 "ultra-low" (and maybe mention that it is for unit testing and debugging), and ne30 "lowres".